documentation/DevelopmentProcess/DevelopmentProcess.tex

```diff
@@ -178,8 +178,12 @@ \subsection{Setup}
 \end{itemize}
 
 \subsection{Versioning Scheme}
-Whenever an item needs to have versions, it will follow the \textit{major.minor.patch} versioning scheme. This means when anything major has changed, such as a feature being added or response to a specification being changed it will increment the current \textit{major} number by one. \textit{Minor} updates will be for changing the item to refine a feature or to meet a specification and will also be incremented by one. For any bug fixes or commentary changes, the \textit{patch} can be incremented by one.
+When items of this system are added/revised, the version number of the system will update. This number will follow the \textit{major.minor.patch} versioning scheme (for example, version 2.1.4). %an example of some sort to make it more clear? Iunno
```

Comment:
Can we switch up added/revised to "added or revised", and "update" to "be updated". I don't think an example really adds more clarity, and would consider removing. With that, removing comments and this can definitely be merged in 👍

Reply:
Yes agreed.

Reply:
Are you gonna make the change?

Reply:
I will right now.